### DISCUSSION TITLE
[New Feature] New key to disallow Mobile Repair Vehicles from repairing a deployed unit

### DISCUSSION THREAD

**Discussion by Bittah** [Vinifera-Developers/Vinifera#1303] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

The only key that prevents a Mobile Repair Vehicle from repairing a structure that has the `UndeploysInto=` key is `ConstructionYard=yes`, but since this key has other side-effects we need a different key for this.

### Proposed Code (optional)

`RepairableByUnit=`

**Comment by FS-21** [comment]
Just play with Ares armors & warheads.
